Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/RoleService.java

```diff
@@ -29,13 +29,15 @@
 import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.RolePreference;
 import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.UserLogin;
 import gov.hhs.fha.nhinc.admingui.services.persistence.jpa.entity.UserRole;
+
+import java.io.Serializable;
 import java.util.List;
 
 /**
  *
  * @author jasonasmith
  */
-public interface RoleService {
+public interface RoleService extends Serializable{
```

Comment:
I committed this before I restarted eclipse after making the formatter preferences update.  I will make the change manually for the next commit.